compiled_starters/cpp/spawn_redis_server.sh

```diff
@@ -6,6 +6,7 @@
 #
 # DON'T EDIT THIS!
 set -e
-cmake . >/dev/null
-make >/dev/null
-exec ./server "$@"
+# vcpkg & cmake are required. 
+# cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=/vcpkg/scripts/buildsystems/vcpkg.cmake
+# cmake --build ./build
```

Comment:
@ryan-gang how will this work if a person tries to run their code locally?

Reply:
Yes, that part is missing. 
I thought they could run the build process themselves. 
Else, I would have to install vcpkg, cmake, run vcpkg install, all the steps. 
It wouldn't be very robust I think.

Reply:
How do you think we'd communicate this? We need an article like https://docs.codecrafters.io/challenges/language-support/java.

Reply:
I don't think we have to install all those tools (we don't do that for any other language), but maybe we can add steps to invoke them like we do for Java / Kotlin? 

I don't think we need to do vcpkg either -- just `cmake` (whatever they'd have to run every time code change).